profiles/pci/ai_sdk/profiles.toml

```diff
@@ -19,3 +19,14 @@ packages = 'cuda cudnn nccl python-pytorch-opt-cuda ollama-cuda tensorflow-opt-c
 device_name_pattern = '(AD|GV|TU|GA|GH|GM|GP)\w+'
 #device_ids = '*'
 
+[rocm-ai-sdk]
+desc = 'AMD AI SDK and related tools'
+nonfree = true
```

Comment:
Is this really nonfree?

Reply:
no, its free. its more to simplify things on the installer side